doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -855,7 +855,10 @@ A subscription is represented by a JSON object with the following fields:
 + `id`: Subscription unique identifier. Automatically created at creation time.
 + `subject`: It is an object that describes the subject of the subscription.
 + `notification`: It is an object that describes the notification received by the subscriber.
-+ `duration`: Duration of the subscription in ISO8601 format. Infinite if not specified.
++ `expires`: Subscription expiration date in ISO8601 format. Permanent subscriptions must omit
+   this field.
```

Comment:
ISO8601 is a very wide format.
We will only accept exact date, including hour, minute and seconds, and ONLY UTC is to be used. Right?
This is very important information and must be documented. 

Emphasize on that it is a **date**, not a duration.

For example, "P1M" is ISO8601 but will not be accepted.